### ISSUE TITLE
WCAG 2.2 rules should be off by default

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3705]
The new target-size rule should be off by default, at least until axe-core 4.6. This should also be clear from the rule-description.md file, which currently lists target-size as a WCAG 2.1 rule.

**Comment by padmavemulapati** [comment]
Validated with the latest axe-core develop branch code base,
for the test script:
```

```
Previously we were seen `target-size` failure for the above code, now as it is off , unable to see that rule listed.
Previous screenshot:


![Image](https://user-images.githubusercontent.com/47409579/195559744-72a2fc02-452a-43be-b6c1-7e06f1ddd359.png)

Now current code is not showing `target-size` rule



![Image](https://user-images.githubusercontent.com/47409579/195560062-63d3f5a5-179d-4714-96c9-9c62c4cc60fb.png)

**Comment by badtant** [comment]
when will target-size be enabled by default?

**Comment by badtant** [comment]
Ok. What is stopping from enabling 2.2 as default?

**Comment by badtant** [comment]
Any updates on this? I would love 2.2 to be default enabled.

**Comment by WilcoFiers** [comment]
No update yet.

**Comment by badtant** [comment]
What is stopping from having 2.2 enabled by default?

**Comment by WilcoFiers** [comment]
Most organizations simply haven't made the switch to WCAG 2.2 yet. The switch to WCAG 2.1 went relatively quickly because it was immediately pulled into EN 301 549. That hasn't happened for WCAG 2.2 yet. I think once 301 549 v 4.1.1 comes out later this year we'll start seeing a shift, at which point we'll probably make the change in axe-core.

**Comment by badtant** [comment]
Hi,
I think EN 301 549 v4.1.1 is released now. Does that mean its time to make 2.2 enabled by default in axe?

**Comment by WilcoFiers** [comment]
@badtant We talked this through internally last week. I was thinking it might be, but our analytics and audit data suggests people are mostly still using WCAG 2.1. If anything % of people using 2.2 went down a bit. I don't know how fast adoption of 301 v4 will be. We're going to wait and see, and take another look next year. 2.2 adoption seems nowhere near as fast as 2.1 adoption was.
